templates/cws/cws_deployment.tmpl

```diff
@@ -0,0 +1,228 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: portal-x
+    app.kubernetes.io/name: cws-test
+  name: cws-test
+  namespace: {{ .Namespace }}
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: portal-x
+      app.kubernetes.io/name: cws-test
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: portal-x
+        app.kubernetes.io/name: cws-test
+        app: cws-test
```

Comment:
this is duplicate of ` app.kubernetes.io/name: cws-test` you can remove

Reply:
If I remove this line, the LoadBalancer service (at the bottom) selector doesn't work.

Reply:
so you update the service side